trunk/cli/src/commands/install.rs

```diff
@@ -28,7 +28,7 @@ pub struct InstallCommand {
 }
 
 #[derive(thiserror::Error, Debug)]
-pub enum PgxInstallError {
+pub enum InstallError {
```

Comment:
In this file we make two changes:
- handle pathed installs
- check for architecture at the level of the whole artifact instead of each file